src/main/java/uk/gov/hmcts/reform/migration/ccd/CoreCaseDataService.java

```diff
@@ -47,14 +49,14 @@ public CaseDetails update(String authorisation, String eventId,
                     .summary(eventSummary)
                     .description(eventDescription)
                     .build()
-            ).data(caseDetails.getData())
+            ).data(updatedCaseDetails.getData())
```

Comment:
Longer term worth replacing this with the `DataMigrationService` `migrate(updatedCaseDetails.getData())` (to save services that little effort of adding it in themselves when they take the template, if they choose to use the approach of implementing in the tool rather than callbacks), but for the purpose of this 👍

Reply:
I think this is needed now - at the moment i cannot see any call to DataMigrationService::migrate anywhere except in tests so implementing a migration according to the Readme wouldn't work - my migrate method would never be run.

Reply:
I have incorporated call to DataMigrationService.migrate() into CoreCaseDataService.update()